app/views/includes/two_thirds_form.html

```diff
@@ -20,6 +20,7 @@
                 <input type="hidden" name="_csrf" value="{{csrfToken}}">
                 {% block form_content %}{% endblock %}
             </form>
+            {% block sign_out %}<a href="/sign-out">Save and close</a>{% endblock %}
```

Comment:
Can we call this block `save_and-close` rather than `sign_out` as it will keep it consistent with the function of the link?